src/App.js

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
-import logo from './logo.svg';
+
 import './App.css';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
+        <img src={require("./jersey.jpg")} width="50%" alt="logo" />
```

Comment:
Use the `import` style like the original logo. Apply the styles for the logo to the `App-logo` css class.

Reply:
Fixed it...